resolver/type_syntax.cc

```diff
@@ -280,7 +280,10 @@ ParsedSig TypeSyntax::parseSig(core::MutableContext ctx, ast::Send *sigSend, con
                     break;
                 }
                 case core::Names::implementation()._id:
-                    sig.seen.implementation = true;
+                    if (auto e = ctx.state.beginError(send->loc, core::errors::Resolver::ImplementationDeprecated)) {
+                        e.setHeader("Use of `{}` is deprecated; please use `{}` instead", "implementation", "override");
```

Comment:
Rather than say "Is deprecated" I would just say "Has been replaced by" (or something else to make it clear that `override` is actually giving them the same benefits as `implementation` used to, and not that something about abstract method checking has gone away).